docs/topics/braintree.rst

```diff
@@ -3,4 +3,23 @@
 Braintree
 #########
 
-More to come.
+Generate a token
+----------------
+
+Calls braintree `ClientToken.generate <https://developers.braintreepayments.com/javascript+python/reference/request/client-token/generate>`_:
+
+.. http:post:: /braintree/token/generate/
+
+    **Request**
```

Comment:
Right. I'm pretty sure we'll want to do saved cards though -- we may have to switch to the drop-in for that. I suppose it should be a buyer uuid.